README.md

```diff
@@ -88,6 +116,86 @@ To deploy PromDash behind a reverse proxy you can set a global path prefix
 using the environment variable `PROMDASH_PATH_PREFIX`. Once set all URLs will
 start with the given prefix.
 
+One simple way to secure your Prometheus deployment with authentication is
+running a reverse proxy, Prometheus, and PromDash on the same machine. The goal
+is to have a publicly available deployment for authenticated users via your
+reverse proxy while restricting raw `IP:PORT` access.
+
+As PromDash executes Prometheus queries directly from the client it is simpler
+to use a single domain. This makes the server side ACL management simpler and
+the client requests already authenticated. 
+
+**Reverse proxy** (e.g. [nginx](http://nginx.org/), [HAProxy](http://www.haproxy.org/))
+* Set up single domain.
+ * Route `http(s)://dash.example.com` to backend `localhost:3000`
+ * Route `http(s)://dash.example.com/prom` to backend `localhost:9090`
+* Configure the same ACL, e.g. HTTP basic auth for both backends.
+
+**Run Prometheus** with binding only to localhost to restrict direct external
+access over the network that would circumvent your authentication. Remember to
+pass the path portion in `-web.external-url`. Example for Docker:
+([full reference](http://prometheus.io/docs/introduction/install/#using-docker))
+
+    docker run -p 127.0.0.1:9090:9090 -v /prometheus-data \
+           prom/prometheus -config.file=/prometheus-data/prometheus.yml \
+                           -web.external-url "http://dash.example.com/prom"
+
+**Run PromDash** in a similar fashion. Once PromDash is running add a
+Prometheus server with `http://dash.example.com/prom/` in the web UI at
+`http://dash.examples.com`.
+
+    docker run -p 127.0.0.1:3000:3000 -v /tmp/prom:/tmp/prom \
+           -e DATABASE_URL=sqlite3:/tmp/prom/file.sqlite3 \
+           prom/promdash
+
+Finally check your reverse proxy is working as expected by visiting
+`dash.examples.com` and `dash.example.com/prom`. Both should prompt you for
+credentials, e.g. HTTP basic auth. You can use a Chrome incognito tabs to force
```

Comment:
I put "tabs" instead of "tab" because incognito wont re-show the prompt. I actually tested this now and it wont even promp if you use a new tab. You have to close all incognito windows then pop up a new one to test `/prom`. The problem is that if you first check `/` and it prompts, then `/prom` it will not prompt, this might be hiding the fact that the route in reverse proxy is not using auth.

Maybe I should remove this whole section and let for the user to figure out. Explaining all this will get kind of long. Not explaining it might leave you vulnerable if you screwed up the routing, though the fact we have the configs now should help out there.

I'll change it to "tab" as "tabs" did not have any real value as I expected.